packages/mobile/src/components/core/SearchInput.tsx

```diff
@@ -28,11 +29,13 @@ const useStyles = makeStyles(({ typography, palette, spacing }) => ({
   }
 }))
 
-type SearchInputProps = TextInputProps
+type SearchInputProps = TextInputProps & {
+  Icon?: React.FC<SvgProps>
```

Comment:
ComponentType<SvgProps> I believe is the best here, since React.FC has implicit children prop